src/index.html

```diff
@@ -10,7 +10,39 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page">
+    <form
+      data-qa="big"
+      class="search"
+      action="/"
+      method = "post"
+    >
+      <div class="search__bar-container">
+        <div class="search__icon"></div>
```

Comment:
if you're adding the icon as a separate div block, it's better to add it to the img tag